shared/src/main/kotlin/org/jetbrains/kotlin/konan/util/Util.kt

```diff
@@ -59,3 +62,69 @@ fun String.removeSuffixIfPresent(suffix: String) =
     if (this.endsWith(suffix)) this.dropLast(suffix.length) else this
 
 fun <T> Lazy<T>.getValueOrNull(): T? = if (isInitialized()) value else null
+
+
+
+/// we use the fake delimiter to support the feature
+/// of the [@Argument] parser to merge several arguments
+/// into the same argument (it works only for [Array<String>]
+/// type, but we do not need the separator
+///
+/// the right fix is to update the [@Argument] to support paths separation
+const val ARGUMENT_NO_DELIMITER = "\\n\\t\\t\\n\\t\\t\\n\\ue000\\ue001\\ue002\\n\\t\\t\\t\\t\\n"
```

Comment:
Nope, this is not the way I believe this feature be implemented. As Slava mentioned, please just add `linkerOpt` and `linkerOpts` arguments for CLI, and concatenate them + implement "\ " escaping for " "-separated options in .def files.